manager/gui/src/components/alerts/LrAlert.vue

```diff
@@ -121,23 +126,16 @@ export default class LrAlert extends Vue {
     }
   }
 
-  setSeverityFilter() {
-    if (this.severity) {
-      let filterList: SgtFilterObject[] = [];
-      const advanceForm = [...this.alertConst.searchConfig.advanceForm];
-      const updatedAdvanceForm = advanceForm.map((element) => {
-        if (element.name === "severity") {
-          element.value = this.severity;
-          filterList.push({
-            label: element.label,
-            name: element.name,
-            value: element.value,
-          });
-        }
-        return element;
-      });
-      this.alertConst.searchConfig.advanceForm = updatedAdvanceForm;
-    }
+  setFilter(filterName: string, filterValue: string) {
+    const advanceForm = [...this.alertConst.searchConfig.advanceForm];
+    const updatedAdvanceForm = advanceForm.map((element) => {
+      if (element.name === filterName) {
```

Comment:
need to modify this for non-editable field, and also use find rather than map.

Reply:
Added a new property 'editable' to handle disabling inputs. 'map' function is replaced with 'findIndex'.